nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py

```diff
@@ -124,121 +135,233 @@ def get_metadata(self):
                 for group_name in unique_elec_group_names
             ],
             Electrodes=[
-                dict(
-                    name="group_name",
-                    description="""The name of the ElectrodeGroup this electrode
-                                is a part of.""",
-                    data=[f"Group{x}" for x in elec_group_names],
-                )
+                dict(name="group_name", description="The name of the ElectrodeGroup this electrode is a part of.")
             ],
             ElectricalSeries_raw=dict(name="ElectricalSeries_raw", description="Raw acquisition traces."),
         )
 
         return metadata
 
 
+class AxonaUnitRecordingExtractorInterface(AxonaRecordingExtractorInterface):
+    """Primary data interface class for converting a AxonaRecordingExtractor"""
+
+    RX = se.AxonaUnitRecordingExtractor
+
+    @classmethod
+    def get_source_schema(cls):
+        return dict(
+            required=["filename"],
+            properties=dict(
+                filename=dict(type="string"),
```

Comment:
why aren't you adding noise_std as an optional argument, or automatically generating this from the init?

Reply:
Good point! That's better!

Reply:
@sbuergers It looks like you didn't make any changes here, you just marked it as resolved

Reply:
@bendichter same issue as before, I did change it, this seems outdated. The changes can be seen here: https://github.com/catalystneuro/nwb-conversion-tools/blob/axona_fixes/nwb_conversion_tools/datainterfaces/ecephys/axona/axonadatainterface.py